src/main/java/lv/ctco/javaschool/auth/boundary/AuthenticationApi.java

```diff
@@ -59,10 +59,12 @@ public Response login(UserLoginDto userLogin, @Context HttpServletRequest reques
     public Response register(UserLoginDto userLogin, @Context HttpServletRequest request, @Context HttpServletResponse response) {
```

Comment:
Tests for `AuthenticationApi`?

Reply:
AuthenticationApiTest - added.
Cchecks only method added to Login&Register template